components/03-modules/gallery/_gallery.scss

```diff
@@ -0,0 +1,102 @@
+$gallery__stage-margin-bottom      : $spacer--medium !default;
+$gallery__nav-vertical-margin-right: 40px !default;
+$gallery__thumb-spacing            : $spacer--medium !default;
+$gallery__thumb-border             : none !default;
+$gallery__thumb-border--active     : 1px solid $green !default;
+$gallery__thumb-width              : 48px !default;
+$gallery__thumb-width--large       : 80px !default;
+$gallery__thumb-arrow-width        : 48px !default;
+$gallery__thumb-arrow-height       : 48px !default;
+$gallery__icon-arrow-width         : 12px !default;
+
+.gallery {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    flex-direction: column;
+    &--vertical {
```

Comment:
move modifiers to the end of class

Reply:
again

Reply:
as there was some disccussion in the past modif should be close to the element they modify? they are in the end of styles. Later we have other elements

Reply:
ok